libkol/Familiar.py

```diff
@@ -18,15 +18,15 @@ def __getitem__(self, key: Union[int, str]):
         future = loop.create_future()
 
         async def getitem():
+            result = None
```

Comment:
There's no need for this in python since it is definitely set in the next conditional. In fact, it messes with mypy a bit (I think, haven't checked)

Reply:
I was about to say it throws without it, but just realised the reason it was throwing that it the previous structure meant that it was trying to add a result after adding an exception, which has now been resolved, so yeah, that's now wholly unnecessary. Should I strip that out now, or is a new branch preferred, now this one is merged?